build/packer/teamcity-agent.sh

```diff
@@ -32,25 +30,26 @@ apt-get update --yes
 # Install the sudo version patched for CVE-2021-3156
 apt-get install --yes sudo
 
-# Install the necessary dependencies. Keep this list small!
-GO_VERSION=1.15
-
 apt-get install --yes \
+  curl \
   docker-ce \
   docker-compose \
   gnome-keyring \
   gnupg2 \
   git \
-  golang-${GO_VERSION}-go \
   openjdk-11-jre-headless \
   pass \
   unzip
 
-# golang-X.Y-go does not install system wide symlinks, it's only done by the
-# golang-go package which points to the latest version. Explicitly symlink the
-# pinned version to /usr/bin.
-for f in go gofm; do
-    ln -s /usr/lib/go-${GO_VERSION}/bin/$f /usr/bin
+curl -fsSL https://dl.google.com/go/go1.15.10.linux-amd64.tar.gz > /tmp/go.tgz
+sha256sum -c - <<EOF
+4aa1267517df32f2bf1cc3d55dfc27d0c6b2c2b0989449c96dd19273ccca051d /tmp/go.tgz
+EOF
+tar -C /usr/local -zxf /tmp/go.tgz && rm /tmp/go.tgz
+
+# Explicitly symlink the pinned version to /usr/bin.
+for f in go gofmt; do
+    ln -s /usr/local/go/bin/$f /usr/bin
```

Comment:
is it worth doing something like
```
for f in `ls /usr/local/go/bin`; do 
   ....
done
```
or adding /usr/local/go/bin to the path directly?

Reply:
Sure. (Not updating the `PATH` -- seems more likely to break something.)